tests/e2e/vsphere_volume_expansion.go

```diff
@@ -717,21 +716,6 @@ var _ = ginkgo.Describe("Volume Expansion Test", func() {
 		gomega.Expect(err).NotTo(gomega.HaveOccurred())
 		gomega.Expect(pvclaim).NotTo(gomega.BeNil())
 
-		ginkgo.By("File system resize should not succeed Since SPS service is down. Expect an error")
```

Comment:
If you delete these lines 720 - 734 , The regular tests which runs in BR pipeline will fail,  
Make a check , If it is Supervisor and if CNS_NEW_SYNC feature is enabled , then only expect the Volume expansion to complete without SPS service